src/bin/parser.js

```diff
@@ -288,8 +276,8 @@ export default class Harmonic {
             await mkdirpAsync(path.join(this.sitePath, 'public', postPath));
```

Comment:
It would be nice if post directory path was put in a variable, and the post filepath be computed from it. E.g.:

```js
const postDirPath = path.join(this.sitePath, 'public', postPath);
const postFilePath = path.join(postDirPath, 'index.html'); // (used to be postSrc)
```